File: js/src/BluefinTecsMerchantPortalError.js


class BluefinTecsMerchantPortalError extends Error {

  isBluefinTecsMerchantPortalError = true

  sdk = 'BluefinTecsMerchantPortal'

  constructor(code, msg, ctx) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

module.exports = {
  BluefinTecsMerchantPortalError
}

File: dart/lib/BluefinTecsMerchantPortalError.dart
class BluefinTecsMerchantPortalError extends Error {
  final bool isBluefinTecsMerchantPortalError = true;

  final String sdk = 'BluefinTecsMerchantPortal';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinTecsMerchantPortalError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinTecsMerchantPortalError: ' + code + ': ' + message;
}
